deploy/recipes/ecfeed/1.0.0/test_package/example.cpp

```diff
@@ -0,0 +1,6 @@
+#include "ecfeed.hpp"
+#include <iostream>
+
+int main() {
+    ecfeed::test();
```

Comment:
do not create a function in ecfeed.hpp to be used solely to test the package. use something like:
```
template_type type = template_type::csv;
std::cout << template_type_url_param(type) << std::endl;
```

Reply:
Done